examples/main-gen.c

```diff
@@ -55,6 +56,9 @@ int main(int argc, char *argv[])
 		char *val;
 	} opt_claims[100];
 	memset(opt_claims, 0, sizeof(opt_claims));
+	char* opt_json;
+	opt_json = (char *) malloc(3*sizeof(char));
+	strcpy(opt_json, "{}");
```

Comment:
Set this to `opt_json = NULL` so we only create this when it's passed on the command line.